circle.yml

```diff
@@ -2,7 +2,7 @@ machine:
   environment:
     DOCKER_IMAGE_NAME: prom/node-exporter
     QUAY_IMAGE_NAME: quay.io/prometheus/node-exporter
-    DOCKER_TEST_IMAGE_NAME: quay.io/prometheus/golang-builder:1.6-base
+    DOCKER_TEST_IMAGE_NAME: quay.io/prometheus/golang-builder:1.7-base
```

Comment:
I updated promu[0] to 1.7.4, but we need to update the quay.io packages.  I'm not sure how to do that.

[0]: https://github.com/prometheus/promu/pull/54